eras/conway/impl/src/Cardano/Ledger/Conway/Governance/DRepPulser.hs

```diff
@@ -94,6 +95,7 @@ data PulsingSnapshot era = PulsingSnapshot
   { psProposals :: !(StrictSeq (GovActionState era))
   , psDRepDistr :: !(Map (DRep (EraCrypto era)) (CompactForm Coin))
   , psDRepState :: !(Map (Credential 'DRepRole (EraCrypto era)) (DRepState (EraCrypto era)))
+  , psPoolDistr :: !(Map (KeyHash 'StakePool (EraCrypto era)) (IndividualPoolStake (EraCrypto era)))
```

Comment:
We should produce the actual stake amount as `Coin`, but it will require some computation, so we should make the field lazy in order to avoid this redundant computation since it is not needed under normal operation.
```suggestion
  , psPoolDistr :: Map (KeyHash 'StakePool (EraCrypto era)) Coin
```